rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -294,8 +293,9 @@ def resize_mirror_normalize(*inputs, bytes_per_sample_hint=0, resize_min=0, resi
             mirror = b.CreateIntParameter(1)
 
     # pybind call arguments
-    kwargs_pybind = {"input_image0": inputs[0], "resize_min":resize_min, "resize_max":resize_max, "mean":mean, "std_dev":std,
-                     "is_output": False, "mirror": mirror}
+    kwargs_pybind = {"input_image0": inputs[0],  "dest_width:" : resize_x , "dest_height": resize_y, "mean":mean, "std_dev":std, "is_output": False,
+                     "scaling_mode": scaling_mode, "max_size": max_size, "resize_shorter": resize_shorter, "resize_longer": resize_longer, "interpolation_type":interpolation_type, "mirror": mirror,
```

Comment:
Either add space between : or not consistently for all params.

Reply:
done